crates/relayer/src/keyring/errors.rs

```diff
@@ -21,9 +21,13 @@ define_error! {
             [ DisplayOnly<anyhow::Error> ]
             |_| { "invalid mnemonic" },
 
-        PrivateKey
+        Secp256k1PrivateKey
```

Comment:
Couldn't have these two as one variant, `InvalidPrivatekey` introduce the type as `{key_type: String}`?